### PR TITLE
(XS) Event picker for adding posts

### PR DESCRIPTION
**Pull request by cmei4444**
Better UI - Creates an easier way to assign an event to a post than a text field, displays event IDs and names in a dropdown menu.

### CODE REVIEW

File: pageserve/templates/index.html [hunk 1]
```diff
@@ -33,8 +33,8 @@ <h2>test user auth</h2>
 <h2>Past posts</h2>
 {% for post in posts %}
 <p>Posted by {{post.author_id}} at {{post.created_at}}</p>
-
-<p>For event {{ post.event_id }}</p>
+<!-- TODO(cmei4444) - link to event info -->
+<p>For event with ID {{ post.event_id }}</p>
```
Comment (evankanderson) [inline pageserve/templates/index.html]: Do you want to include this as a link instead?

Comment (cmei4444) [reply pageserve/templates/index.html]: I wanted to include an option to open up details for a selected event on a separate page so someone could check that they were posting to the right event. 
It's not a super important feature so I was going to leave it to another PR if I had time.

Comment (evankanderson) [reply pageserve/templates/index.html]: SGTM.

**Comment by evankanderson** [comment]
LGTM

**Pull request merged**